src/App.js

```diff
@@ -59,9 +51,14 @@ const App = () => {
     .then(() => readBookClubs())
     .catch((error) => console.log("New Book Club created error:", error))
   }
-
-
-
+  useEffect (() => {
+    const loggedIn = localStorage.getItem("currentUser")
+    if (loggedIn) {
+      setCurrentUser(JSON.parse(loggedIn))
+    }
+    readBookClubs();
+    readMemberships();
+  }, [])
```

Comment:
I got to research this...but I think that the useEffect() needs to be higher up...Suggestion. is for the readBookClubs(), readMemberships(), and useEffect() functions to be higher up. I think it impacts the reloading of the page.